flutter_app/pubspec.yaml

```diff
@@ -23,7 +23,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  app_settings: 4.0.4
```

Comment:
So its not a major issue now but just remember to keep things consistent. Either remove the ^ from all your imports or keep it

Reply:
understood, like the idea of removing them so i will change it.

Reply:
resolved